core/src/main/java/io/questdb/griffin/engine/functions/date/TimestampFloorOffsetFunctions.java

```diff
@@ -48,7 +48,6 @@ public AbstractTimestampFloorOffsetFunction(Function arg, int stride, long offse
             this.arg = arg;
             this.stride = stride;
             this.offset = offset;
```

Comment:
there is now unused code in this class

Reply:
It was always there, seems that it can be useful some day.

Reply:
i removed it, as it turns out we had two classes like that, useful function lives in Timestamps. It is still used